src/Sql/Sql/help/Remove-AzSqlInstance.md

```diff
@@ -98,7 +98,7 @@ Parameter Sets: RemoveInstanceFromInputParameters
 Aliases: InstanceName
 
 Required: True
-Position: 0
+Position: 1
```

Comment:
@johnpaulkee same comment about reversing the position of `-Name` and `-ResourceGroupName`

Reply:
Acked.